src/translations.js

```diff
@@ -52,9 +56,14 @@ module.exports = class Translation {
         };
       } else {
         // If cache cannot be used and no previous cache available download , update cache and serve it.
-        let cembraLocales=["ch","it","fr","en"];
+        let cembraLocales=["de","it","fr","en"];
```

Comment:
same here

Reply:
yes, the downloaded file has the name "de.json" so i am saving it in cache based on file name as  "de" .